packages/edit-site/src/components/page-pages/index.js

```diff
@@ -39,19 +44,74 @@ const defaultConfigPerViewType = {
 	},
 };
 
-export default function PagePages() {
+const PATH_TO_TYPE = {
+	'/pages': 'page',
+};
+
+function useView() {
 	const {
-		params: { path, activeView = 'all' },
+		params: { path, activeView = 'all', isCustom = 'false' },
 	} = useLocation();
-	const initialView = DEFAULT_VIEWS.find(
-		( { slug } ) => slug === activeView
-	).view;
-	const [ view, setView ] = useState( initialView );
+	const type = PATH_TO_TYPE[ path ];
+	const selectedDefaultView =
+		isCustom === 'false' &&
+		DEFAULT_VIEWS[ type ].find( ( { slug } ) => slug === activeView )?.view;
+	const [ view, setView ] = useState( selectedDefaultView );
+
+	const customViews = useCustomDataViews( type );
```

Comment:
Not sure I understand why we need this hook, specially since we're calling `getEditedEntityRecord` right after.

Reply:
Oh you're using the slug in the url, why not just the id directly? it would simplify a lot of things IMO

Reply:
Updated to use the id directly.